cmd/auth/describe.go

```diff
@@ -0,0 +1,180 @@
+package auth
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/flags"
+	"github.com/databricks/databricks-sdk-go/config"
+	"github.com/spf13/cobra"
+)
+
+var authTemplate = `{{"Workspace:" | bold}} {{.Details.Host}}
+{{if .Username }}{{"User:" | bold}} {{.Username}}{{end}}
+{{"Authenticated with:" | bold}} {{.Details.AuthType}}
+{{if .AccountID }}{{"Account ID:" | bold}} {{.AccountID}}{{end}}
```

Comment:
Template in case of accounts has a newline issue (and says "workspace"):
```
% ./cli auth describe -p deco-azure-prod-acct --account
Workspace: https://accounts.azuredatabricks.net

Authenticated with: azure-cli
Account ID: ...
-----
Configuration:
  ✓ account_id: ...
```